modules.json

```diff
@@ -0,0 +1,486 @@
+{
+  "tenpin":
+  [
+    {
+      "name": "accelerometer",
+      "package": "accel-mma84",
+      "chip": "MMA8452Q",
+      "datasheet": "http://www.freescale.com/files/sensors/doc/data_sheet/MMA8452Q.pdf",
+      "image": "https://s3.amazonaws.com/technicalmachine-assets/product+pics/2014+05+15+production+modules/accelerometer.jpg",
```

Comment:
You may want to have more than one image for a given module, with meta data about each.

Recommend: 

```js
"images": [ 
  { "description": "Front view", "src": "https://s3.amazonaws.com/technicalmachine-assets/product+pics/2014+05+15+production+modules/accelerometer.jpg" },
  { "description": "Rear view", "src": "rear.jpg" },
]
```